pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
     "setuptools",
     "setuptools_scm>=6.2",
-    "cython==0.29.34",
+    "cython==0.29.36",
```

Comment:
Is this really needed for gzip to work?

Reply:
not for gzip, but needed for me to get astropy's extensions compile at all under 3.12rc1